x-pack/plugins/cloud_security_posture/public/pages/findings/findings_table.tsx

```diff
@@ -23,7 +23,7 @@ import * as TEXT from './translations';
 import type { CspFinding } from './types';
 import { CspEvaluationBadge } from '../../components/csp_evaluation_badge';
 import type { FindingsGroupByNoneQuery, CspFindingsResult } from './use_findings';
-import { FindingsRuleFlyout } from './findings_flyout';
+import { FindingsRuleFlyout } from './findings_flyout/findings_flyout';
```

Comment:
`/findings_flyout` would be the first folder in `findings`

i actually put the all flat on purpose to avoid too many folders with one file in them. 

IMO we should revert this.